app/static.json

```diff
@@ -0,0 +1,7 @@
+{
+  "root": "build/",
+  "clean_urls": false,
+  "routes": {
+    "/**": "index.html"
+  }
+}
```

Comment:
Out of curiosity, what does this do? I can't seem to find much about it, I'm guessing it's for Heroku?

Reply:
Tiny thing but heads up that the commit "added static.json file to fix routing issues on heroku" doesn't include `static.json`, but there's a bunch of other files unrelated to that commit message.  

I'm guessing in the moving process something happened where it thought that it got deleted. I found that when I move files, it looks like a file was deleted and new one added, but by putting them in the same commit, then it shows it was just moved, preserving any commit history from before.

Reply:
In the future it might be easier to separate those out into their own commits. It means more commits, but makes it easier to understand a file's history 😸

Reply:
I was trying out a solution but worked through something with George and posted in H1!